### DISCUSSION TITLE
Add multiple folders to container

### DISCUSSION THREAD

**Discussion by shandshellin** [jlesage/docker-freefilesync#22] [General]
How would i go about adding anoither folder to this container so i could browse source and destination for the sync? Just add it with whatever name and it will populate to acess it?

**Comment by chuanchuan199310** [comment]
可以在docker compose 中直接添加 volumes 做本地文件夹的映射，
/vol2/1000/docker:/docker # docker容器文件夹
/vol1/1000/media:/media # 影视文件夹
